ReactAndroid/src/main/java/com/facebook/react/animated/DecayAnimation.java

```diff
@@ -44,8 +48,15 @@ public void runAnimationStep(long frameTimeNanos) {
         (1 - Math.exp(-(1 - mDeceleration) * (frameTimeMillis - mStartFrameTimeMillis)));
 
     if (Math.abs(mLastValue - value) < 0.1) {
-      mHasFinished = true;
-      return;
+
+      if (mNumLoops == -1 || mCurrentLoop < mNumLoops) { // looping animation, return to start
+        mStartFrameTimeMillis = -1;
```

Comment:
Why -1 here? Should it be `mStartFrameTimeNanos = frameTimeNanos;` like in the FrameBasedAnimationDriver?